tests/python/driver/tvmc/test_compiler.py

```diff
@@ -65,7 +67,7 @@ def test_cross_compile_aarch64_tflite_module(tflite_mobilenet_v1_1_quant):
 
     graph, lib, params, dumps = tvmc.compiler.compile_model(
         tflite_mobilenet_v1_1_quant,
-        target="llvm -device=arm_cpu -mtriple=aarch64-linux-gnu -mattr=+neon",
+        target="llvm -device=arm_cpu -mtriple=aarch64-linux-gnu -mattr='+neon'",
```

Comment:
Dont you think the user should be able to provide compiler arguments with or without quotes when they only have a single element ? I think I understand the requirement comes because mattr here expects a list. However, I think it could be something handled by the parser.

Reply:
Yes, I agree that's sensible, and will revisit the parser to cope with that.

Reply:
The latest version of this PR covers the suggestion here, plus added unit testing.